src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -1,296 +1,314 @@
+                                       [data]="queriesResults.READIOPS"
+                                       [data2]="queriesResults.WRITEIOPS">
+              </cd-dashboard-area-chart>
+              <cd-dashboard-area-chart chartTitle="OSD Latencies"
+                                       dataUnits="ms"
+                                       decimals="2"
+                                       label="Apply"
+                                       label2="Commit"
+                                       [data]="queriesResults.READLATENCY"
+                                       [data2]="queriesResults.WRITELATENCY">
+              </cd-dashboard-area-chart>
+              <cd-dashboard-area-chart chartTitle="Client Throughput"
+                                       dataUnits="B/s"
+                                       decimals="2"
+                                       label="Reads"
+                                       label2="Writes"
+                                       [data]="queriesResults.READCLIENTTHROUGHPUT"
+                                       [data2]="queriesResults.WRITECLIENTTHROUGHPUT">
+              </cd-dashboard-area-chart>
+              <cd-dashboard-area-chart chartTitle="Recovery Throughput"
+                                       dataUnits="B/s"
+                                       decimals="2"
+                                       label="Recovery Throughput"
+                                       [data]="queriesResults.RECOVERYBYTES">
+              </cd-dashboard-area-chart>
+            </div>
+          </cd-card>
+        </div>
       </div>
-    </cd-card>
+    </div>
   </div>
 </div>
 
 <ng-template #alertsCard>
-  <ng-container *ngFor="let alert of alerts; let i = index">
-    <div [ngClass]="borderClass"
-         *ngIf="alertType === alert.labels.severity">
+  <ng-container *ngFor="let alert of prometheusAlertService.alerts; let i = index">
+    <div [ngClass]="['border-'+alertClass[alert.labels.severity]]"
+         *ngIf="alert.labels.severity === alertType || !alertType">
       <div class="card tc_alerts border-0 pt-3">
-        <div class="row no-gutters">
+        <div class="row no-gutters ps-2">
           <div class="col-sm-1 text-center">
-            <span [ngClass]="[icons.stack, icons.large, textClass]">
+            <span [ngClass]="[icons.stack, icons.large, 'text-'+alertClass[alert.labels.severity]]">
               <i [ngClass]="[icons.circle, icons.stack2x]"></i>
               <i [ngClass]="[icons.stack1x, icons.inverse, icons.warning]"></i>
             </span>
           </div>
-          <div class="col-md-11">
-            <div class="card-body ps-0 pe-1 pt-1">
+          <div class="col-md-10 ps-0">
+            <div class="card-body ps-0 pe-1 pb-1 pt-0">
               <h6 class="card-title bold">{{ alert.labels.alertname }}</h6>
               <p class="card-text me-3"
                  [innerHtml]="alert.annotations.summary"></p>
```

Comment:
Lets show description of alert instead of summary.
```suggestion
                 [innerHtml]="alert.annotations.description"></p>
```

Reply:
hmm, summary would be better for an overview page as description is more descriptive and doesn't look nicer with the view. 
![Screenshot from 2023-09-26 21-28-33](https://github.com/ceph/ceph/assets/71764184/ec16a7f5-774f-48f9-ad3e-c8656c526573)

Reply:
The problem I see with Summary is it doesn't adds value, the messages in summary are pretty vague.

IMO nicer view vs value addition, I will choose both. What we can do is to add text ellipsis if the text is too huge by setting the max height if each alert row. If you think it doesn't look good.

I think it just looks fine with large test as well.